backend/oasst_backend/api/v1/tasks.py

```diff
@@ -9,12 +10,36 @@
 from oasst_backend.tree_manager import TreeManager
 from oasst_shared.exceptions import OasstError, OasstErrorCode
 from oasst_shared.schemas import protocol as protocol_schema
+from oasst_shared.schemas.protocol import User
 from sqlmodel import Session
 from starlette.status import HTTP_204_NO_CONTENT
 
 router = APIRouter()
 
 
+@dataclass
+class TaskContext:  # todo: come up with a better name?
+    pr: PromptRepository
+    tm: TreeManager
```

Comment:
same as "pr", as defined in [PEP 8: Function and Variable names](https://peps.python.org/pep-0008/#function-and-variable-names), it should contain the whole name with underscores

pr should be prompt_repository
tm should be tree_manager